ui/admin/tests/unit/controllers/scopes/scope/edit-test.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+import { module, test } from 'qunit';
+import { setupTest } from 'ember-qunit';
+
+module('Unit | Controller | scopes/scope/edit', function (hooks) {
```

Comment:
I am wondering if we can verify that we have correctly defined `scopes` in the controller even if there is nothing else to test for this specific controller. Thoughts on that?

Reply:
of course, i can do something like assert.ok(controller.scopes) perhaps?

Reply:
I think that should work. Honestly it is just a double check to make sure it is accessible at that controller level. The controller (scopes in this case) doesn't need to be tested since it should already have it's own unit tests.

Reply:
alrighty, thank you for the feedback

Reply:
updated